### PR TITLE
Improve infra-check workflow to not fail but comment about the non-ifra files

### PR DESCRIPTION
**Pull request by jkonecny12**
Most of the infrastructure PRs are failing now which is wrong. Let's improve the state by instead of fail just comment with a message that these files should be double checked.

2022-02-04T14:13:15,581244481+01:00

### CODE REVIEW

**Review by VladimirSlavik — APPROVED** [review]
Thank you, this will be very useful.

**Review by poncovka — APPROVED** [review]
Thank you!

File: .github/workflows/infra-check.yml [hunk 1]
```diff
@@ -64,12 +67,14 @@ jobs:
             done
 
             if [ $matched == "no" ]; then
-              echo "$f is not part of the infrastructure"
+              echo "$f" >> ./message.txt
```
[the remark this answers is not in the document]
Comment (jkonecny12) [reply .github/workflows/infra-check.yml]: Fixed, thanks for pointing that out.

**Review by rvykydal — APPROVED** [review]

**Pull request merged**